.eslintrc.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
         sourceType: 'module',
     },
     ignorePatterns: [
+        '!.*',
```

Comment:
I dont get which files this pattern matches in the fui core. Or: What happens if that line is not added?
 Could you please explain?

Reply:
files starting with `.` are not fixed by eslint by default, this way they are, so the configs itself are

Reply:
Ah, ok, thanks 👍🏼